### PR TITLE
feat(Apps/CI): Add check for broadcast_text

### PR DESCRIPTION
**Pull request by Kitzunu**
## Changes Proposed:
-  why not be too sure that people don't fuck with the table unless they know what they are doing :D

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

File: apps/ci/ci-pending.sh [hunk 1]
```diff
@@ -31,5 +31,16 @@ for i in `find data/sql/updates/pending* -name "*.sql" -type f`; do
     fi
 done
 
+for i in `find data/sql/updates/pending* -name "*sql" -type f`; do
+    if $(cat "$i"|sed "s/'.*'\(.*\)/\1/g"|grep -q -i -E "broadcast_text"); then
+        echo "> broadcast_text check - OK"
+    else
```
Comment (Kitzunu) [inline apps/ci/ci-pending.sh:37]: could someone just proof check that I have the if/else in the right order and that the regex is correct? :D

Comment (Winfidonarleyan) [reply apps/ci/ci-pending.sh:37]: just add test file

Comment (Kitzunu) [reply apps/ci/ci-pending.sh:37]: True. Didnt think of that hahaha

**Comment by Kitzunu** [comment]
ok works

**Review by Winfidonarleyan — APPROVED** [review]

**Review by Nyeriah — APPROVED** [review]

**Pull request merged**